packages/fxa-auth-server/lib/senders/emails/templates/cadReminderFirst/ar.ftl

```diff
@@ -1,7 +1,3 @@
-# This Source Code Form is subject to the terms of the Mozilla Public
-# License, v. 2.0. If a copy of the MPL was not distributed with this
-# file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
 # We have kept this one ftl file in a different locale just for Storybook setup
```

Comment:
Haven't checked into your PR, but do we need to do anything here to _not_ include this one since it's for Storybook?

This is looking great though👍

Reply:
Per our Slack chat, going to remove this for now as it won't work properly with the new setup. But I have an idea for down the road :)